vmtkApps/CerebralAneurysms/ParentVesselReconstruction/clipvoronoidiagram.py

```diff
@@ -270,13 +272,13 @@ def SmoothClippedVoronoiDiagram(voronoi,centerlines):
 originalVoronoiDiagram = ReadPolyData(voronoiDiagramFilename)
 patchCenterlines       = ReadPolyData(patchCenterlinesFilename)
 
-print 'Clipping Voronoi Diagram'
+print('Clipping Voronoi Diagram')
 maskedVoronoiArray    = MaskVoronoiDiagram(originalVoronoiDiagram,patchCenterlines)
 clippedVoronoiDiagram = ExtractMaskedVoronoiPoints(originalVoronoiDiagram,maskedVoronoiArray)
 WritePolyData(clippedVoronoiDiagram, clippedVoronoiDiagramFilename)
 
 if (smoothVoronoiDiagram==1):
-   print 'Smoothing Voronoi Diagram',
+   print('Smoothing Voronoi Diagram', end=' ') # end seperator new in py3, will throw error in py2 environments
```

Comment:
---
@lantiga reply
---

  Preview

We don't need `end=' '` here. Line 279 in the original code just had a training comma.